<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QMainWindow" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>668</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>668</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>768</width>
     <height>668</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>768</width>
     <height>668</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>768</width>
        <height>668</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>768</width>
        <height>668</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget{
	background-image: url(:/src/GUI/bgmain.png);	
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>9</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
    image: url(:/src/GUI/close.png);
    border-style: outset;
}
QPushButton:hover{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
QPushButton:pressed{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="minimizeButton">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
    image: url(:/src/GUI/min.png);
    border-style: outset;
	border: none;
}
QPushButton:hover{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
	border: none;
}
QPushButton:pressed{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
	border: none;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>24</number>
             </property>
             <property name="topMargin">
              <number>24</number>
             </property>
             <property name="rightMargin">
              <number>24</number>
             </property>
             <property name="bottomMargin">
              <number>24</number>
             </property>
             <item>
              <widget class="QLabel" name="usernameLabel">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(112, 112, 112);
border:none;</string>
               </property>
               <property name="text">
                <string>InkosiZhong</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="moreButton">
               <property name="font">
                <font>
                 <family>Roboto</family>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton{
    image: url(:/src/GUI/more.png);
    border-style: outset;
	border: none;
}
QToolButton:hover{
    image: url(:/src/GUI/more_hover.png);
    border-style: outset;
	border: none;
}
QToolButton:pressed{
    image: url(:/src/GUI/more.png);
    border-style: outset;
	border: none;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLineEdit" name="friendSearchEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	border: none;
}
QLineEdit:focus{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	border: 2px solid rgb(90, 147, 255);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="friendScrollArea">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: outset;</string>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>250</width>
                <height>479</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QRadioButton{
	color: rgb(112, 112, 112);
	background-color: rgb(240, 240, 240);
	spacing: 85px;
}
QRadioButton:hover{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	spacing: 85px;
}
QRadioButton:press{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	spacing: 100px;
}
QRadioButton:indicator{
	width: 0px;
	height: 0px;
	border: none;
}</string>
                 </property>
                 <property name="text">
                  <string>Mr.Chang</string>
                 </property>
                 <property name="icon">
                  <iconset resource="src.qrc">
                   <normaloff>:/src/GUI/notice.png</normaloff>:/src/GUI/notice.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QRadioButton{
	color: rgb(112, 112, 112);
	background-color: rgb(240, 240, 240);
	spacing: 100px;
}
QRadioButton:hover{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	spacing: 100px;
}
QRadioButton:press{
	color: rgb(112, 112, 112);
	background-color: rgb(219, 219, 219);
	spacing: 100px;
}
QRadioButton:indicator{
	width: 0px;
	height: 0px;
	border: none;
}</string>
                 </property>
                 <property name="text">
                  <string>Mr.Gao</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <property name="leftMargin">
              <number>24</number>
             </property>
             <property name="topMargin">
              <number>12</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="friendNameLabel">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(112, 112, 112);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="leftMargin">
              <number>24</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>24</number>
             </property>
             <item>
              <widget class="QTextEdit" name="msgShowTextEdit">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(240, 240, 240);
border:none;</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="leftMargin">
              <number>24</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextEdit" name="msgSendTextEdit">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>120</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(254, 254, 254);
border:none;
color: rgb(112, 112, 112);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="font">
                <font>
                 <family>Roboto</family>
                </font>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="sendFileButton">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#sendFileButton{
 image: url(:/src/GUI/icon/send_file.png);
 background-color: rgb(254, 254, 254);
 border:none;
}
#sendFileButton:hover{
 image: url(:/src/GUI/icon/send_file_hover.png);
 background-color: rgb(254, 254, 254);
 border:none;
}
#sendFileButton:pressed{
 image: url(:/src/GUI/icon/send_file_press.png);
 background-color: rgb(254, 254, 254);
 border:none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="whisperCheckBox">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator{
 	border:none; 
 	width: 25px;
	height: 25px;
}
QCheckBox::indicator:checked{
	image: url(:/src/GUI/icon/whisper_hover.png);
}
QCheckBox::indicator:enabled:unchecked{
	image: url(:/src/GUI/icon/whisper.png);
}
QCheckBox::indicator:enabled:unchecked:hover{
	image: url(:/src/GUI/icon/whisper_hover.png);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendPushButton">
               <property name="minimumSize">
                <size>
                 <width>131</width>
                 <height>54</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>131</width>
                 <height>54</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border-image: url(:/src/GUI/button.png);
	color:rgb(254, 254, 254);
}
QPushButton:hover{
	border-image: url(:/src/GUI/button_hover.png);
	color:rgb(254, 254, 254);
}
QPushButton:pressed{
	border-image: url(:/src/GUI/button_press.png);
	color:rgb(254, 254, 254);
}
</string>
               </property>
               <property name="text">
                <string>Send</string>
               </property>
               <property name="icon">
                <iconset resource="src.qrc">
                 <normaloff>:/src/GUI/send.png</normaloff>:/src/GUI/send.png</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+Return</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
